Makefile

```
@@ -57,13 +57,13 @@ install-protoc:
 	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31
 
 install-golangcilint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.59
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.62.2
 
 ensure_go_version:
-	@go version | grep -q 'go1.22' || (echo "Please use go1.22" && exit 1)
+	@go version | grep -q 'go1.23' || (echo "Please use go1.23" && exit 1)
 
-ensure_golangcilint_1_59:
-	@golangci-lint --version | grep -q '1.59' || (echo "Please use golangci-lint 1.59" && exit 1)
+ensure_golangcilint_1_62_2:
```

Does this pattern serve a purpose? Or could we drop it?
```suggestion
ensure_golangcilint_version:
```

---

should drop it